src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
This is doable in newer version of JS but in this projects version this does not work. Probably you couldn't pass the test. You should try to change it.

Reply:
@ysrckr ok, got it !